README.md

```diff
@@ -82,6 +82,17 @@ max_line_length=off
 # Note that rules in any ruleset other than the standard ruleset will need to be prefixed 
 # by the ruleset identifier.
 disabled_rules=no-wildcard-imports,experimental:annotation,my-custom-ruleset:my-custom-rule
+
+# Defines the imports layout. There are predefined layouts like "ascii" or "idea", as well as a custom layout.
```

Comment:
Nice documentation :+1:. Probably should mention that this will be available starting Kotlin 1.4-M3 release.

Reply:
But I mean it's available for ktlint already now, so you can specify the custom layout for ktlint, there's just no such capability in Kotlin IDEA plugin yet.. or do you want to also mention about the kotlin plugin just in case?

Reply:
You are right, we could mention it later when it will be publicly available.